suite-common/wallet-utils/src/__tests__/accountUtils.test.ts

```diff
@@ -337,4 +340,113 @@ describe('account utils', () => {
         account.addresses = enhanceAddresses(accountInfo, account);
         expect(account.addresses.change).toEqual([]);
     });
+
+    it('isDebugOnlyAccountType', () => {
+        expect(isDebugOnlyAccountType('legacy', 'btc')).toBe(false);
+        expect(isDebugOnlyAccountType('segwit', 'btc')).toBe(false);
+        expect(isDebugOnlyAccountType('coinjoin', 'btc')).toBe(false);
+        expect(isDebugOnlyAccountType('taproot', 'btc')).toBe(false);
+        expect(isDebugOnlyAccountType('ledger', 'btc')).toBe(false);
+        expect(isDebugOnlyAccountType('legacy', 'eth')).toBe(true);
+        expect(isDebugOnlyAccountType('ledger', 'eth')).toBe(true);
+    });
+
+    const accounts = [
+        getWalletAccount({ symbol: 'eth', accountType: 'legacy' }),
+        getWalletAccount({ symbol: 'eth', accountType: 'normal' }),
+        getWalletAccount({ symbol: 'eth', accountType: 'ledger' }),
+        getWalletAccount({ symbol: 'btc', accountType: 'coinjoin' }),
+        getWalletAccount({ symbol: 'btc', accountType: 'taproot' }),
+        getWalletAccount({ symbol: 'btc', accountType: 'legacy' }),
+        getWalletAccount({ symbol: 'btc', accountType: 'segwit' }),
+        getWalletAccount({ symbol: 'btc', accountType: 'ledger' }),
+        getWalletAccount({ symbol: 'matic', accountType: 'legacy' }),
+        getWalletAccount({ symbol: 'matic', accountType: 'normal' }),
+        getWalletAccount({ symbol: 'matic', accountType: 'ledger' }),
+        getWalletAccount({ symbol: 'sol', accountType: 'normal', empty: true, visible: false }),
+        getWalletAccount({ symbol: 'sol', accountType: 'ledger' }),
+        getWalletAccount({
+            symbol: 'sol',
+            accountType: 'ledger',
+            empty: true,
+            visible: false,
+        }),
+    ];
+
+    type RunFilterReceiveAccountsTestParams = {
+        isDebug?: boolean;
+        receiveNetwork?: string;
+        deviceState?: `${string}@${string}:${number}`;
+    };
+
+    const runFilterReceiveAccouns = ({
```

Comment:
is it safe to create such complex helper method for tests? what if there is bug too @Lemonexe @peter-sanderson

Reply:
I think it's necessary, but I didn't add anything new to it, just copied the behavior I've seen in useCoinmarketVerifyAccount

<img width="728" alt="image" src="https://github.com/user-attachments/assets/f314b50a-4c1d-4b4d-b91f-f386d8d75aa5">